IZ14307: parallel build failure when building res files.
Patch taken from cws_srx644_ause05

Index: oo_cvs/solenv/inc/target.mk
===================================================================
RCS file: /cvs/tools/solenv/inc/target.mk,v
retrieving revision 1.135
retrieving revision 1.135.2.1
diff -u -3 -p -r1.135 -r1.135.2.1
--- oo_cvs/solenv/inc/target.mk	24 Apr 2003 13:00:31 -0000	1.135
+++ oo_cvs/solenv/inc/target.mk	15 May 2003 14:53:58 -0000	1.135.2.1
@@ -2327,6 +2327,10 @@ $(MISC)$/$(TARGET)_xxl_%.done : %.xxl
 .INCLUDE : tg_rslb.mk
 .ENDIF
 .ENDIF
+
+.IF "$(RSC_MULTI1)$(RSC_MULTI2)$(RSC_MULTI3)$(RSC_MULTI4)$(RSC_MULTI5)$(RSC_MULTI6)$(RSC_MULTI7)$(RSC_MULTI8)$(RSC_MULTI9)"!=""
+$(RSC_MULTI1) $(RSC_MULTI2) $(RSC_MULTI3) $(RSC_MULTI4) $(RSC_MULTI5) $(RSC_MULTI6) $(RSC_MULTI7) $(RSC_MULTI8) $(RSC_MULTI9): $(IMGLSTTARGET)
+.ENDIF          # "$(RSC_MULTI1)$(RSC_MULTI2)$(RSC_MULTI3)$(RSC_MULTI4)$(RSC_MULTI5)$(RSC_MULTI6)$(RSC_MULTI7)$(RSC_MULTI8)$(RSC_MULTI9)"!=""
 
 # -------
 # - SHL -
